ocfweb/stats/printing.py

```diff
@@ -152,8 +153,7 @@ def _pages_per_day():
         ''')
 
         last_seen = {}
-        pages_printed = {}
-
+        pages_printed = pages_printed = defaultdict(partial(defaultdict, int))
```

Comment:
It looks like you accidentally assigned here twice, unless I am mistaken.

Also if you don't mind, a comment explaining why we have a `defaultdict(partial(defaultdict, int))` would be appreciated. Thanks!

Reply:
Ah, good catch on the double assignment, I was clearly looking too fast and didn't see that

Agreed on a comment

Reply:
Oops! thanks for the reviews, double assignment should be fixed. I need sleep :p

Reply:
Happens to everyone :P